tools/cli/commands.go

```diff
@@ -886,7 +881,7 @@ func DescribeWorkflow(c *cli.Context) {
 // DescribeWorkflowWithID show information about the specified workflow execution
 func DescribeWorkflowWithID(c *cli.Context) {
 	if !c.Args().Present() {
-		ExitIfError(errors.New("workflow_id is required"))
+		ErrorAndExit("Parameter workflow_id is required.", nil)
```

Comment:
Better to consist with Argument instead of Parameter

Reply:
Updated. Will send out a new PR about the change.